tools/osbuild-mpp

```diff
@@ -811,6 +880,9 @@ class ManifestFileV2(ManifestFile):
         files = element_enter(self.sources, "org.osbuild.curl", {})
         self.source_urls = element_enter(files, "items", {})
 
+        inline_files = element_enter(self.sources, "org.osbuild.inline", {})
+        self.inline_files = element_enter(inline_files, "items", {})
```

Comment:
This will always create an (mostly empty) org.osbuild.inline node in the preprocessed json. I think we should call element_enter lazily when needed, or e.g. the osbuild test files will break due to this unnecessary change.